helpers.lua

```diff
@@ -244,8 +244,17 @@ function helpers.sysctl_async(path_table, parse)
 
     spawn.with_line_callback("sysctl " .. path, {
         stdout = function(line)
+	    local separators = {
+	       openbsd = "=",
```

Comment:
Vicious uses 4 spaces for indentation.  I assume you uses emacs which somehow always mess up the indentation level somewhere around 3 or 4 spaces :stuck_out_tongue:

Reply:
Addressed in commit. Thanks! :)